pkgs/data/fonts/ohxproto/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "0xproto";
+  version = "1.300";
+  underscoreVersion = builtins.replaceStrings [ "." ] [ "_" ] version;
+
+  src = fetchzip {
+    url = "https://github.com/0xType/0xProto/releases/download/${version}/0xProto_${underscoreVersion}.zip";
+    sha256 = "sha256-RanIMf9P2lFOF3kJS6jMlh/X6jttofbHSqFUJxWSqKk=";
+  };
+
+  dontConfigure = true;
+  dontPatch = true;
+  dontBuild = true;
+  dontFixup = true;
+  doCheck = false;
```

Comment:
Maybe we could avoid disabling all of these phases? I think it's better to leave them as they are in case anyone needs to override them. AFAIK, they shouldn't mess up anything, unless there are any broken tests (but that isn't the case here right?).

Reply:
I've tested it and everything works fine if I remove this. I only disabled them because they weren't used and saw it in another similar derivation here. Will remove this.